dataflow/run-inference/Dockerfile

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
```

Comment:
No need for this.

Reply:
Whoops, thanks for catching. I must have had copied-pasted the license from a Python file :)